infrastructure/lib/constructs/authentication.ts

```diff
@@ -108,9 +108,14 @@ export interface CognitoAuthenticationProps {
 
 function samlAttributeMapping(props: CognitoAuthenticationProps): { [key: string]: string } {
   return {
-    email: "email",
-    family_name: "lastName",
-    given_name: "firstName",
+    email: "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress",
+    family_name: "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname",
+    given_name: "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname",
+    phone_number: "telephone",
+    "custom:dod_id": "employeeID",
+    "custom:rank": "rank",
+    "custom:citizenship": "citizenship",
+    "custom:branch": "branch",
```

Comment:
How do you feel about the LOE to make these attributes passed in to `idpProps` to the AtatWebApiStack? It can get added to the tech debt list. But maybe it's fine if we consider these just won't be mapped if they aren't in the assertion.